nixos/modules/services/web-apps/alps.nix

```diff
@@ -0,0 +1,99 @@
+{ lib, pkgs, config, ... }:
+
+with lib;
+
+let
+  cfg = config.services.alps;
+in {
+  options.services.alps = {
+    enable = mkEnableOption "alps";
+
+    port = mkOption {
```

Comment:
This too can be simplified as a single option `listenAddress`, which allows for neat shorthand like `:8080`.

Reply:
* The "neat shorthand" will AFAIU only appear in the end-result, so it's not that interesting, isn't it?
* I think it's a reasonable use-case to change the listenIP while wanting to keep the default port, hence two options.

Reply:
I also prefer to split it into options so its easier to use from the user site. Less opportunity to miss configure.